bindings/wasm/examples/worker.ts

```diff
@@ -12,791 +12,55 @@
-  'add',
-  'subtract',
-  'intersect',
-  'decompose',
-  'warp',
-  'transform',
-  'translate',
-  'rotate',
-  'scale',
-  'mirror',
-  'calculateCurvature',
-  'calculateNormals',
-  'smoothByNormals',
-  'smoothOut',
-  'refine',
-  'refineToLength',
-  'refineToTolerance',
-  'setProperties',
-  'setTolerance',
-  'simplify',
-  'asOriginal',
-  'trimByPlane',
-  'split',
-  'splitByPlane',
-  'slice',
-  'project',
-  'hull'
-];
-// CrossSection static methods (that return a new cross-section)
-const crossSectionStaticFunctions = [
-  'square', 'circle', 'union', 'difference', 'intersection', 'compose',
-  'ofPolygons', 'hull'
-];
-// CrossSection member functions (that return a new cross-section)
-const crossSectionMemberFunctions = [
-  'add', 'subtract', 'intersect', 'rectClip', 'decompose', 'transform',
-  'translate', 'rotate', 'scale', 'mirror', 'simplify', 'offset', 'hull'
-];
-// top level functions that construct a new manifold/mesh
-const toplevelConstructors = ['show', 'only', 'setMaterial'];
-const toplevel = [
-  'setMinCircularAngle', 'setMinCircularEdgeLength', 'setCircularSegments',
-  'getCircularSegments', 'resetToCircularDefaults', 'Mesh', 'GLTFNode',
-  'Manifold', 'CrossSection', 'setMorphStart', 'setMorphEnd'
-];
-const exposedFunctions = toplevelConstructors.concat(toplevel);
-
-// Setup memory management, such that users don't have to care about
-// calling `delete` manually.
-// Note that this only fixes memory leak across different runs: the memory
-// will only be freed when the compilation finishes.
-
-const memoryRegistry = new Array<Manifold|CrossSection>();
-
-function addMembers(
-    className: string, methodNames: Array<string>, areStatic: boolean) {
-  //@ts-ignore
-  const cls = module[className];
-  const obj = areStatic ? cls : cls.prototype;
-  for (const name of methodNames) {
-    const originalFn = obj[name];
-    obj[name] = function(...args: any) {
-      //@ts-ignore
-      const result = originalFn(...args);
-      memoryRegistry.push(result);
-      return result;
-    };
-  }
-}
-
-addMembers('Manifold', manifoldMemberFunctions, false);
-addMembers('Manifold', manifoldStaticFunctions, true);
-addMembers('CrossSection', crossSectionMemberFunctions, false);
-addMembers('CrossSection', crossSectionStaticFunctions, true);
-
-for (const name of toplevelConstructors) {
-  //@ts-ignore
-  const originalFn = module[name];
-  //@ts-ignore
-  module[name] = function(...args: any) {
-    const result = originalFn(...args);
-    memoryRegistry.push(result);
-    return result;
-  };
-}
-
-// Debug setup to show source meshes
-let ghost = false;
-const shown = new Map<number, Mesh>();
-const singles = new Map<number, Mesh>();
-
-const FPS = 30;
-
-const GLOBAL_DEFAULTS = {
-  roughness: 0.2,
-  metallic: 1,
-  baseColorFactor: [1, 1, 0] as [number, number, number],
-  alpha: 1,
-  unlit: false,
-  animationLength: 1,
-  animationMode: 'loop'
+evaluator.context = {
+  ...evaluator.context,
```

Comment:
If this line was forgotten, would everything break? Perhaps we should have something like `evaluator.addFunctions()` instead?

Reply:
If this was omitted, the evaluator would still include all of the manifold functions, but not the additional context specified here.

Reply:
Even if just line 32 was gone but the rest of the expression remained? What is in that existing context that you're unpacking and repacking?

Reply:
Oh, I see!  Originally I had split this up into successive updates.  Here, the destructuring is vestigial, and I'll get rid of it.

Reply:
I added some accessor methods to evaluate while I was at it.